src/main/java/org/openstreetmap/atlas/utilities/testing/OsmosisXmlReaderFromResource.java

```diff
@@ -0,0 +1,162 @@
+package org.openstreetmap.atlas.utilities.testing;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.logging.Level;
```

Comment:
Do we want to update this to be slf4j for consistent logging across code base or does something break if we do this?

Reply:
So I copied all this from the osmosis code base and chose to change only the ingest so it can read from a resource. That import came from osmosis. I'd rather not change what was not strictly needed

Reply:
Fair enough. Just curious though, does the logging work properly if something is broken?